core/src/test/java/org/apache/iceberg/TestOffsetsBasedSplitScanTaskIterator.java

```diff
@@ -25,26 +25,39 @@
 import org.junit.Test;
 
 public class TestOffsetsBasedSplitScanTaskIterator {
+
```

Comment:
Nit: unless this was required by lint checks, could we avoid adding newlines? They can cause conflicts.